llvm/include/llvm/CodeGen/MachineInstr.h

```diff
@@ -117,6 +117,7 @@ class MachineInstr
     NoConvergent = 1 << 17,  // Call does not require convergence guarantees.
     NonNeg = 1 << 18,        // The operand is non-negative.
     Disjoint = 1 << 19,      // Each bit is zero in at least one of the inputs.
+    DefinesFP = 1 << 20,     // Defines floating-point value
```

Comment:
This does not belong as an instruction flag

Reply:
Removed the flag